gradle/versions.gradle

```diff
@@ -75,15 +75,17 @@ dependencyManagement {
     dependency 'org.hyperledger.besu:plugin-api:21.1.3'
     dependency 'org.hyperledger.besu.internal:metrics-core:21.1.3'
 
-    dependencySet(group: 'tech.pegasys.teku.internal', version: '20.11.0') {
+    dependencySet(group: 'tech.pegasys.teku.internal', version: '21.6.0') {
       entry 'bls'
-      entry 'core'
-      entry 'datastructures'
+      entry('core') {
+        exclude 'teku:data'
```

Comment:
Why does teku:data need to be excluded?

Reply:
This one is reported by core's pom however it is not published (as it is empty project - contains submodules though). The license report plugin was failing since it can't find the dependency. 

This should be fixed in the latest release of Teku. Let me remove it. (It was added while testing the earlier release 21.5.0).